src/com/company/EvenNumbersGenerator.java

```diff
@@ -0,0 +1,18 @@
+package com.company;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class EvenNumbersGenerator {
+    public List<Integer> getEvenNumbersList(int min, int max) {
+        List<Integer> evenNumbersList = new ArrayList<>();
+        for (int i = min; i <= max; i++)
+        {
+            if(i%2 == 0)
```

Comment:
it is okay, but I'd extract that to a variable so that it is a bit readable, for example 

`if( i % 2  == 0)`

-->

```
boolean isEven = I % 2 == 0;
if (isEven) 
```

Reply:
you can also write it using streams

```
public List<Integer> getEvenNumbersList(int min, int max) {
            IntPredicate isEven = i -> i % 2 == 0;
            return IntStream.range(min, max)
                    .filter(isEven)
                    .boxed()
                    .collect(Collectors.toList());
        }
```

Reply:
streams are totally new for me, at this moment I wouldn't use streams at this place.